build/go-playground.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=go-playground
+After=network.target
+
+[Service]
+Type=notify
+User=go-playground
+WorkingDirectory=/home/go-playground
```

Comment:
Usually third-party software should be placed in `/opt` directory (like `/opt/playground`).

Also nice to have to provide some installation script that will build playground and install systemd unit.
Something like `make install`

Reply:
Ah, those are indeed great ideas. I was indeed just about to see that the `GOROOT` environment variable can be fetched with `go env` during installation so that the unit file is dynamically updated before being installed.